perf/security/sds-tests/egress-sds/README.md

```diff
@@ -0,0 +1,50 @@
+# Test description
+
+This is a TLS egress gateway SDS test.
+The test sets up a number of TLS egress gateways for a group of nginx services.
+The test creates a group of sleep pods, where each sleep sends HTTP requests to an nginx
+service periodically. For example, sleep-1 sends HTTP requests to my-nginx-1,
+and sleep-2 sends HTTPS requests to my-nginx-2. The sleep pods are deployed in clientns namespace and nginx services are deployed
+in a namespace called mesh-external".
+
+## To run the SDS test originating mTLS at gateway
+
+- Create a GKE cluster and set it as the current cluster.
+Here this test is run on the cluster *istio-testing*
+on GCP project *istio-security-testing*.
+You may use `kubectl config current-context` to confirm that your newly created cluster
+is set as the current cluster.
```

Comment:
These description may not be necessary. You can just saying that `Create a GEK cluster and run the test against your created cluster`.